ipa-core/src/ff/boolean_array.rs

```diff
@@ -358,6 +382,20 @@ boolean_array_impl!(
     ]
 );
 
+// impl BA112
+boolean_array_impl!(
+    boolean_array_112,
+    BA112,
+    112,
+    14,
+    [
+        1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
No need for this if we land @martinthomson's crazy metaprogramming thingy first.

Reply:
You'll need `[0,][0,][0,][0,][][0,][0,]` for that.